docker/bacalhau-image/Dockerfile

```diff
@@ -1,10 +1,22 @@
 # syntax=docker/dockerfile:1.4
```

Comment:
kind of, but harmless. There are git commands called as part of `build-bacalhau` that checks if files have changed since the last build before triggering a new build. In our case it will just print these errors, but will continue with the build. will be nice to not show this mess though